Resources/Prototypes/_EE/Entities/Mobs/Customization/Markings/harpy.yml

```
@@ -0,0 +1,179 @@
+      state: colorless_zetawing
+      
+- type: marking
+  id: HarpyLimeZetawing
+  bodyPart: RArm
+  markingCategory: RightArm
+  speciesRestriction: [Harpy]
+  coloring:
+    default:
+      type:
+        !type:SimpleColoring
+          color: "#ffffff"
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)